arch/posix/core/irq.c

```diff
@@ -8,6 +8,8 @@
 #include "board_irq.h"
 
 #ifdef CONFIG_IRQ_OFFLOAD
+#include "irq_offload.h"
```

Comment:
why do you use quotation marks and not brackets when including headers that are not in the same directory?

Reply:
well.. not a specially good reason: in case I needed to change header search paths as some point, in the POSIX arch I tended to use the `<>` for the underlying OS headers, and `""´ for Zephyr ones, unless I was adding to a file which already existed, or so, but as I ended up not needing it I was not consistent after.. In Zephyr, do we differentiate "system headers" from "normal headers" or so?. (If we only use the `-I` compiler option an never `-iquote` it makes little difference apart from cosmetics)

Reply:
if you feel more comfy with this changes to <> I can just do it. Would you prefer it that way?

Reply:
it's orthogonal to this patch but it would be nice to address this at some point.